Source/Reporting/Web/src/app/Reporting/CaseReportsForListing/filtering/filter.pipe.ts

```diff
@@ -2,10 +2,10 @@ import {Pipe, PipeTransform} from '@angular/core';
 import {CaseReportForListing} from '../CaseReportForListing';
 
 enum CaseReportStatus {
-    Success = "Success",
-    TextMessageParsingError = "TextMessageParsingError",
-    UnknownDataCollector = "UnknownDataCollector",
-    TextMessageParsingErrorAndUnknownDataCollector = "TextMessageParsingErrorAndUnknownDataCollector"
+    Success = 0,
```

Comment:
I don't know what numbers to use here 🤷‍♂

Reply:
The enum value definitions can be found in the backend c# code with the same names

Look in the Concepts project and the Readmodels project for enum definitions

Reply:
Ok thanks, looks like it was correct